### PR TITLE
Matomo doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: docs/destinations/analytics/matomo.mdx [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+### Web SDK settings
+
+<img src="../../assets/event-stream-destinations/matomo-connection-settings-4.png" alt="Matomo connection settings"/>
+
+- **Use device mode to send events**: As this is a device mode-only destination, this setting is enabled by default and cannot be disabled.
+
+## Identify
+
+The <Link to="https://www.rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-spec/identify/">`identify`</Link> call lets you create new or update existing customers and record the traits about them like their name, email address, etc.
```
Comment (ItsSudip) [inline docs/destinations/analytics/matomo.mdx:76]: We don't create or update the customer data with the identify call. We just set the id value in cache with the userId of identify call. So this information is redundant.

File: docs/destinations/analytics/matomo.mdx [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+```
+
+<div class="infoBlock">
+RudderStack does not consider any parameters inside the  <code class="inline-code">page</code> call and simply calls the  <code class="inline-code">trackPageView</code> event for each  <code class="inline-code">page</code> call.
+</div>
+
+
+## Custom dimensions
+
+Custom dimensions can be used to track any information related to any action or visitors. RudderStack provides `customDimension` field which can be used inside the `integrations` object, as shown below:
```
Comment (ItsSudip) [inline docs/destinations/analytics/matomo.mdx:203]: We should also mention that we can use the custom dimensions through track calls only.

File: docs/destinations/analytics/matomo.mdx [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+"category": "Orders",
+"action": "Pass",
+"value": 15
+});
+```
+
+RudderStack supports `track` calls for **standard** and **e-commerce** events as explained below: 
+
+### Standard events
+
```
Comment (ItsSudip) [inline docs/destinations/analytics/matomo.mdx:108]: By default, we call trackEvent for track call but first, we verify if the event name belongs to the standard event list from the rudderstack dashboard, if it belongs to that list we call that specific event with some specific properties. @rohithkaza can you update the notion card with all the required properties of standard events?

**Comment by ameypv-rudder** [comment]
As we have updated the overall docs architecture, merging this PR would cause a major conflict. Hence, created a new PR [here](https://github.com/rudderlabs/rudderstack-docs/pull/735) and will address all the changes/suggestions/feedback. @rohithkaza @ItsSudip

File: docs/destinations/analytics/matomo.mdx (hunk 3 above)
Comment (rohithkaza) [reply docs/destinations/analytics/matomo.mdx:108]: added

File: docs/destinations/analytics/matomo.mdx [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+description: Step-by-step guide on sending your event data from RudderStack to Matomo.
+---
+
+[Matomo](https://matomo.org/), formerly Piwik, is an open-source web analytics platform. It can be used to derive valuable insights into your website's visitors and marketing campaigns so as to optimize your strategy and online experience for your visitors.
+
+RudderStack supports Matomo as a destination to which you can seamlessly send your event data.
+
+<div class="successBlock">
+Find the open source transformer code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/">GitHub repository</a>.
+</div>
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/destinations/analytics/matomo.mdx:12]: Removed this block.

File: docs/destinations/analytics/matomo.mdx [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+Refer to the <Link to="#faq">FAQ</Link> section below for more information on finding the Server URL and Site Id.
+</div>
+
+### Event settings
+
+<img src="../../assets/event-stream-destinations/matomo-connection-settings-1.png" alt="Matomo connection settings"/>
+<img src="../../assets/event-stream-destinations/matomo-connection-settings-2.png" alt="Matomo connection settings"/>
+<img src="../../assets/event-stream-destinations/matomo-connection-settings-3.png" alt="Matomo connection settings"/>
+
+- **Mapping to trigger the Matomo Goal ID for the respective Events**: Enter the event name and the corresponding goal Id which needs to be mapped.
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/destinations/analytics/matomo.mdx:51]: Made this change and added a callout.

File: docs/destinations/analytics/matomo.mdx [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+- **Mapping to trigger the Matomo standard events for the respective Events**: Enter the event name and pick the corresponding standard event from the dropdown which needs to be mapped.
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/destinations/analytics/matomo.mdx:52]: Made this change and added a callout.

File: docs/destinations/analytics/matomo.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply docs/destinations/analytics/matomo.mdx:76]: Fixed this.

File: docs/destinations/analytics/matomo.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply docs/destinations/analytics/matomo.mdx:203]: Added this info.

File: docs/destinations/analytics/matomo.mdx (hunk 3 above)
Comment (ameypv-rudder) [reply docs/destinations/analytics/matomo.mdx:108]: Will look into this.

**Pull request closed without merging**